composer.json

```diff
@@ -14,8 +14,8 @@
         "issues": "https://github.com/FrozenNode/Laravel-Administrator/issues"
     },
     "require": {
-        "php": ">=5.3.0",
-        "laravel/framework": "4.*",
+        "php": ">=5.4.0",
+        "laravel/framework": "5.*",
```

Comment:
Also add `"illuminate/html": "5.0.*"` as it is no longer included in the `laravel/framework` package.